core/src/test/java/org/elasticsearch/index/query/InnerHitBuilderTests.java

```diff
@@ -312,7 +313,7 @@ public static InnerHitBuilder randomInnerHits(boolean recursive, boolean include
         innerHits.setScriptFields(new HashSet<>(scriptFields.values()));
         FetchSourceContext randomFetchSourceContext;
         if (randomBoolean()) {
-            randomFetchSourceContext = new FetchSourceContext(randomBoolean());
+            randomFetchSourceContext = new FetchSourceContext(true, Strings.EMPTY_ARRAY, Strings.EMPTY_ARRAY);
```

Comment:
Thanks for the comments @jimczi, I'm away from home till after the Easter weekend. Will pick them up when back!